hw2.py

```diff
@@ -0,0 +1,25 @@
+#1.
+a = int(input("What's the starting number?"))
+b = int(input("What's the ending number?"))
+c = int(input("What is the count?"))
+for i in range(a, b, c):
```

Comment:
What happens if b>a?  after reading b, it can be checked if a<b and if not to close program.
What happens if c = 0?
What happens if user enters values different than integer?  https://stackoverflow.com/questions/3501382/checking-whether-a-variable-is-an-integer-or-not check here for some examples to determine if a value is integer